### PR TITLE
Translation Into Russian

### PR DESCRIPTION
**Pull request by Saimon398**
Hello, 
I decided to contribute to the project and have this manual translated into Russian ( #218  ).
Submitting it for review)

### CODE REVIEW

File: readme-ru.md [hunk 1]
```diff
@@ -0,0 +1,2190 @@
...
+
+С другой стороны, руководства по "классическим снимкам" призывают хранить большие файлы (например, разметку рендеринга компонента, результат API JSON) на каком-то внешнем носителе и каждый раз при выполнении теста сравнивать полученный результат с сохраненной версией. Это может привести к тому, что, тест будет привязан к огромному количеству данных (например, 1000 строк), о которых разработчик никогда не задумывался. Почему так нельзя делать? аким образом, существует 1000 причин, по которым ваш тест может не пройти - достаточно изменения одной строки, чтобы снимок стал недействительным, а это, скорее всего, будет происходить часто. Как часто? Каждый пробел, комментарий или незначительное изменение HTML/CSS. Мало того, что название теста не дает представления о том, что пошло не так, поскольку он просто проверяет, что 1000 срок кода остались неизменны, так еще и вводит разработчика в заблуждение, заставляя принимать за желаемую истину документ, который он не сможет проверить. Все это является признаками теста, который стремиться захватить сразу многое.
```
Comment (Patr56) [inline readme-ru.md:481]: typo: "? аким образом"?

**Comment by goldbergyoni** [comment]
@Saimon398 WOW, this is freakin awesome!

First, sorry for the very late reply, I was away from OSS work for a few months due to some circumstances

This PR is gorgeous, many-many thanks for this

Don't forget to also link to this from the international homepage, this is how Russian folks will find this

I'll try to promote this as much as possible - Twitter, news section in my repos, etc

Is there a Russian FB/Whatsapp group where we can publish this?

🎁 One last note - I'm now finalizing a 7-hours long comprehensive, advanced Node.js testing course - I'd like to gift to you as my way of saying that you for this awesome work. I'll bounce in a month+

@all-contributors please add @Saimon398 for content

**Comment by goldbergyoni** [comment]
https://github.com/all-contributors please add @Saimon398 for content

**Comment by goldbergyoni** [comment]
@all-contributors please add @Saimon398 for content

**Pull request merged**